### PR TITLE
Error logging 31

### PR DESCRIPTION
**Pull request by ypenn21**
Cloud Functions error logging: https://github.com/google/quota-monitoring-solution/issues/31

### CODE REVIEW

File: quota-notification/src/main/java/functions/ConfigureAppAlertHelper.java [hunk 1]
```diff
@@ -276,7 +288,8 @@ private static String createOrUpdateNotificationChannel(AppAlert appAlert){
         notificationChannel = updateNotificationChannel(client, appAlert);
       }
     } catch (IOException e) {
-      logger.severe("Can't create Notification channel "+e.toString());
+      MDC.put("severity", "CRITICAL");
```
Comment (ypenn21) [inline quota-notification/src/main/java/functions/ConfigureAppAlertHelper.java:291]: MDC is to add the severity field back in cloud logging after switching to using log back https://medium.com/google-cloud/java-logging-on-cloud-run-with-stackdriver-9786d6fdbe17

File: quota-scan/pom.xml [hunk 2]
```diff
@@ -51,6 +51,21 @@
 
   <dependencies>
     <!-- Required for Function primitives -->
+    <dependency>
```
Comment (ypenn21) [inline quota-scan/pom.xml:54]: dependencies for logstash and log back

File: terraform/example/new-policy.yaml
Comment (bgood) [inline terraform/example/new-policy.yaml:1]: I think we need to remove this file from the PR.

File: terraform/modules/qms/main.tf [hunk 3]
```diff
@@ -575,6 +576,16 @@ resource "google_logging_project_sink" "instance-sink" {
   depends_on             = [module.project-services]
 }
 
+resource "null_resource" "update_logging_sink" {
+  triggers = {
+    always_run = "${timestamp()}"
+  }
+
+  provisioner "local-exec" {
+    command = "gcloud logging sinks update _Default --log-filter=\"NOT LOG_ID('cloudaudit.googleapis.com/access_transparency') AND NOT LOG_ID('externalaudit.googleapis.com/access_transparency') AND NOT (severity=DEFAULT OR severity=DEBUG AND resource.labels.function_name=quotaMonitoringListProjects) AND NOT (severity=DEFAULT OR severity=DEBUG AND resource.labels.function_name=quotaMonitoringScanProjects)\""
```
Comment (bgood) [inline terraform/modules/qms/main.tf:585]: Why is access transparency included in the exclusion or some examples of what would be excluded by these conditions?

**Review by bgood — CHANGES_REQUESTED** [review]

**Review by bgood — APPROVED** [review]

**Pull request closed without merging**